notebook/Solution/02 - Autoregressive Integrated Moving Average (ARIMA).ipynb

```diff
@@ -0,0 +1,1776 @@
W4+FwV/Hino2LX1Yztf2AprzSvyf6I5S7rJQ5S2/+oFCcihJ3xYxzMhDlWM+w2yQb5VFM7fNiONg5hNkkWFY7Ompk+TwvnUMxBoqMmHmrK4gQw2GGk6XCbcPnsHBct9ynazNVoVDirphxXjqsNWu57BytDEBWOI1yzRzoGGJxlbtgxcZV9VonnP0dQ0WNdbgrxIIKF07b1OqCCCFoqvbo4h6Zts1UhUKJu2LGefFwL7U+ey4ee2GlG7NJGLapeqAjWNAlA+SKfe1rHyxqrLe6glN2yWRZXOXmUFeQ9kBUWe6KaUOJu2JGyWQkLx/pZfPS4R6YNouJhRUuQyz3wUiSk4HomOJe4bZRV+Zgf/vElnsyneF4b7ige2cyNFW56QnGyUhYaEAmqkJRCCXuihllX/sQA5Ekb1s2MkxxsUERMwc6NdFeUTe2tb2yzleUW6a5N0wqIznHAHHPMtUm2ArFRChxVxTNGy0D7DwxYOiYL+j+9lOTgJbUuGnujZBKZ0oa/0BHVtzHrrW+qt7H0Z4wsWThnjJP7OmgtT/CW12lRcpkyRf3BSrGXTFNKHFXFEUsmebO/9zJP/56j2FjSil59PU2zlvgH1HfHMiVq23Ti3JNlQMdQ1S4bdR4xy55u7LeRzojOdQ5up7N4a4g/+Ph1/ncI29wSI+UKbUQV1bc3TYzVR7juvsoFPkocVcUxaOvt9ETjHOsJzwla7ovFKdzMDaiZ+lrzQMc6wlz+wWj240tyeuUVAoHO4OsqPOO26syGzGzr4Df/UdbmwF4vSXAw6+cYEGFa8w+qcXitluo9dlZUOme1h6airMbJe6KCUmlM/zbn45hMQkS6QzNfZGJLzqFdz2wlYv++VnO+8of+MbvDyGl5JHXWvDYLdy0tm7U+ctqPQhRfIgiaFb6x360nb5QPDfvQ51BVswbv/1dQ7kTr8PC/o6RETOBSIJfvd7Gezc2sKreR184UbJLJsst6+dz45p5hoylUBRCibtiQn63p4OW/gifunwJAEe6J1eONxRP0dIf4epza9i0qILvPHeEe3+7nyf2dHDz+npcttEljnwOK0urPexqDRR9nxfe6uG5Qz38w6/3IKXke88fJZ7KsGmCfqRCCFbW+UZZ7j/f3kosmeGTb2vin25aCcC584wR93+4YQV/edUyQ8ZSKAqhxF0xLlmRXFbj4VNXaOL+1iTS9UGLMgF478YGHvzQRt5/wQJ+vLWZWDLD+88fuwP8hgV+3mgZGOHKGY+Tekek3+/r4guP7uabf3iLd22Yz7Wrxu9tCprf/WBHkHRGu1cyneEn25q5ZEkl587zceHiSn78sfP52OZFRc1FoZhplLgrxuWPB7s52BnkU5cvwWO30FDunHTf0eHStpqP+au3ruaDFy7gmpW1rJ4/tstkw4JyBiLJ3PUT0R6IsrzWy/mLytmyo421DWX887vXFOXXXl1fRjSZzoVf7m4bpGMwxgcuHH74XLG8RvUiVcwZVMlfxZhIKXng+aPM9zu5eX09AOfUejk8yS5JzXr1w2yqvdkkuO9daya8bsMCPwBvtA4U1TC6bSBKY4WTL79zFd997gife/uyojc/1zVq93qzNcA5tV7eaNFCPi+YwKWjUMxWlOWuGJPtx/vZeWKAv7h8ca687bIaz6QjZpp7w9R47biL6FSUz7IaL26bmTdaAkWdfzIQZb7fSWOFi6+9Zy11ZeO3vctncZUbr93Cm23avd5oDTDf76TG5xj/QoVilqLEXTEmDzx/lCqPjds2NeaOLav1kkhnONFffMTMib4Ii6ZQQ8VsEqxr9Bcl7kOxJMFYasI+pmNhMgnWNpbxZqsWMfPGiYHcJweFYi6ixF1RkJ5gnD+91cOHLlo0wrWRTb2fjGvmeF94ytUPNyzwc6BjiGiicPZolpN6stN8/9TT+dc1aPc60RemfTDGhgXlUx5LoZhplLgrCpJNHjpvoX/E8Wx2ZrENLsJxrQl0MT7zQmxoLCeVkeydoGpjTtynaLmD5ndPZSQPv9qi3VtZ7oo5jBJ3RUGyUSOnNqVw2bSImbeKjJjJRrpMxS0DsF4X2NcnqGmTDYOc75+6uK/XN1Uf2d6CzWxiVf34yU8KxWxGibuiIEd7QrhsZurKRm8onlPr5VBncZmjp0bKTJYqj535fmfB0gD5nAxEsVlMVLqnXqul1udgns/BUCzFynofdktpZQYUiplkQnEXQjQKIZ4TQhwQQuwTQnxOP14hhPiDEOKw/r0875q7hRBHhBCHhBDXTucCFNPDke4QS6o9BWPEz19UwVtdIdoDExf1yor7VN0yoGWFHpzgYXJyQIuUMZlKq9WyrlGrM6NcMoq5TjGWewr4WynlCuAi4DNCiJXAXcCzUsplwLP6z+jv3Q6sAq4DHhBCKBNojnGsJ5zrZXoq1+gZn0/v65xwnObeMFUeO55JhkHms3yel2M9YRKp4fDLAx1DfPKh17j+/75IMp2hTQ+DLJX1jZqNojZTFXOdCcVdStkhpXxdfx0EDgDzgVuAh/TTHgJu1V/fAjwipYxLKY8DR4ALDJ63Ai3JqD+coDcUJxRPGTZuJJHiZCA6ZhPoJdUeltZ4+P2+rgnHau6L0FRiQ4rl87ykMjK3D/CDl45zw/0v8sJbvRzoGOKVY305y71UrllVy8aF5WxeUlnyWArFTDIpn7sQYhGwAXgVqJVSdoD2AABq9NPmA615l7Xpx04d604hxA4hxI6enp4pTF3x463NnPeVP7Dpq8+w4X89bVhD6WM9mitlyTh1y69dVcv25n4GwolxxzrRFy65T2i20UbWNfPQ1mY2LijnpS9cictm5rFd7fSG4iVFymRZUu3h0U9fosoMKOY8RYu7EMIDPAr8tZRyPAdoIafnqMpPUsoHpZSbpJSbqquri52GIo+n9nayqNLFl9+5koyEX+5sM2TcsSJl8rl21TzSGcmzB7tHHN97cpD/9dv9bP7aH1nz5d/TNRRn0RQ3U7M0VbmxmgUHO4O
```

Comment:
please add these comment cause sometime the autoarima may not working correctly
import six
import sys
sys.modules['sklearn.externals.six'] = six
import joblib
sys.modules['sklearn.externals.joblib'] = joblib

Reply:
Thanks BK! Works like a charm. Have updated it